test/Microsoft.Azure.Devices.Edge.Test.Common/config/ModuleConfigBuilder.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Config
+{
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class ModuleConfigBuilder : BaseModuleConfigBuilder
```

Comment:
This is confusing. Why do we need both of these? Why can't BaseModuleConfigBuilder not have CreateOptions (as far as I can tell this is the only diff)?

Reply:
``ModuleConfigBuilder` has `status` and `restartPolicy`, which are not needed by edge agent (which is why edge agent's config builder inherits directly from `BaseModuleConfigBuilder` instead of `ModuleConfigBuilder`.) `BaseModuleConfigBuilder` could technically have `createOptions`, but we never set it on edge agent (at least in our tests) so I didn't see the need to add it at that level.

If we want this to more closely match the config spec I can add `createOptions` to the base class, but there's still `status` and `restartPolicy`.